.github/workflows/renovate-deploy.yml

```diff
@@ -11,13 +11,25 @@ permissions: {}
 jobs:
   renovate:
     runs-on: ubuntu-latest
+    outputs:
+      owner: ${{ steps.set_values.outputs.owner }}
+      repositories: ${{ steps.step2.outputs.repositories }}
     steps:
+      - name: Set owner and repositories values
+        id: set_values
+        run: | 
+          owner=$(echo "${{ github.event.inputs.deployRepository }}" | cut -d '/' -f1)
+          repositories=$(echo "${{ github.event.inputs.deployRepository }}" | cut -d '/' -f2)
+          echo "owner=$owner" >> "$GITHUB_OUTPUT"
+          echo "repositories=$repositories" >> "$GITHUB_OUTPUT"
       - name: Get token
         id: get_token
         uses: actions/create-github-app-token@31c86eb3b33c9b601a1f60f98dcbfd1d70f379b4 # v1.10.3
         with:
           app-id: ${{ secrets.RENOVATE_APP_ID }}
           private-key: ${{ secrets.RENOVATE_PRIVATE_KEY }}
+          owner: ${{ steps.set_values.outputs.owner }}
+          repositories: ${{ steps.set_values.outputs.repositories }}
```

Comment:
The `cut` that you did above could be done directly here instead?

Reply:
Nope. 

`with` parameters expect always a string. 

I tried here:
https://github.com/rancher/renovate-config/actions/runs/10178931770/job/28153591846

```
owner and repositories set, creating token for repositories "$(echo "rancher/rke-tools" | cut -d '/' -f2)" owned by "$(echo "rancher/rke-tools" | cut -d '/' -f1)"
```